app/controllers/games_controller.rb

```diff
@@ -1,38 +1,38 @@
 class GamesController < ApplicationController
   before_action :set_game, only: [:show, :edit, :update, :destroy]
-
+  
   attr_accessor :name
-
+  
   def name
     "Game #{self.id}"
   end
-
+  
   # GET /games
   # GET /games.json
   def index
-    # @games = Game.all
+    @games = Game.all.is_available
```

Comment:
You should be able to use the scope instead of `.all` - http://api.rubyonrails.org/classes/ActiveRecord/Scoping/Named/ClassMethods.html#method-i-scope